### DISCUSSION TITLE
Unable to verify student developer pack

### DISCUSSION THREAD

**Discussion by foolaris** [community/community#51256] [GitHub Education]
Hi, First time I submitted my student developer pack application was on March 21st. 

I remember the materials were successfully submitted and the website reminded me that I would receive an email reply with the review result within 4 days. However, it has been a week now and I have not received any response in my email. 

Today, I tried to submit the application again, but I encountered the following error: "Your uploaded proof has already been used previously", which caused the submission to fail. 

So what should I do next? I am looking forward to your answer. Thank you sincerely.

<img width="801" alt="image" src="https://user-images.githubusercontent.com/128502388/228317651-ee46548e-25d2-49bb-964c-444e59149521.png">

**Comment by samus-aran** [comment]
Hi @foolaris 👋 

Here are a few tips to help you when re-evaluating your application for submission.

Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

We would also suggest that when you update your profile information that you log out and log back into GitHub before re-applying.

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

If the system is detecting a similar image of your evidence we would recommend you take a photo of the evidence in a different environment, angle or camera to demonstrate that the latest image used is current. Alternatively you can use another type of evidence in your submission.

Hope this helps! Let us know how you get along 👍
